virttest/virt_vm.py

```diff
@@ -1397,6 +1397,30 @@ def get_cpu_count(self):
         finally:
             session.close()
 
+    def get_online_cpu_count(self):
```

Comment:
this method is exact duplicate of `get_cpu_count()`.

Reply:
instead we can have fix in existing method with `check_cmd` as extra argument that could default to "cpu_chk_cmd",
```python
def get_cpu_count(self, check_cmd="cpu_chk_cmd"):
    session = self.wait_for_login()
    cmd = self.params.get(check_cmd)
    try:
        out = session.cmd_output_safe(cmd)
        return int(re.search("\d+", out, re.M).group())
    finally:
        session.close()
```

Reply:
Thanks, a better way.